Scripts/Build.ps1

```diff
@@ -10,6 +10,7 @@ param (
   # Settings
   [switch]$ci = $false,
   [string]$config = "Debug",
+  [string]$msbuild = "",

```

Comment:
The 17.3 toolset still has the `--standalone` bug. Had to manually use the 17.4 toolset to do the release. Seems valuable so I kept it.